src/mscorlib/corefx/System/Globalization/SortKey.cs

```diff
@@ -52,26 +52,6 @@ internal SortKey(String localeName, String str, CompareOptions options, byte[] k
             _options    = options;
             _string   = str;
         }
-
-        [OnSerializing]
```

Comment:
Unix only, didn't diff those yet. I"ll revert any.

---
In reply to: [99095564](https://github.com/dotnet/coreclr/pull/9269#discussion_r99095564) [](ancestors = 99095564)